builder/installer_utilities.livecodescript

```diff
@@ -13,6 +13,10 @@ constant kRunOnce = "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersio
 local sIsFaceless
 local sFacelessLog
 
+private function isInstallerEnvironment
+   return (the environment begins with "installer")
```

Comment:
This is slightly more efficient 😛 
Although I admit it will be problematic if we ever decide to add an "installerblahblahthisjunkisimportant" environment